assets/src/edit-story/elements/text/output.js

```diff
@@ -62,11 +64,61 @@ function TextOutput({
     ...generatePatternStyles(color, 'color'),
   };
 
+  const highlightStyle = {
+    ...fillStyle,
+    fontSize: undefined,
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    webkitBoxDecorationBreak: 'clone',
```

Comment:
When saving, I get a warning in the console:

```
Warning: Unsupported vendor-prefixed style property webkitBoxDecorationBreak. Did you mean WebkitBoxDecorationBreak?
```

Reply:
Not consistently though. Not sure why it came about.

Reply:
In this case, the `-webkit-box-decoration-break` is not being autogenerated. Anyway, just fixed the `w` lowercase 😄

Reply:
🤔 Looks like that our `npm run lint:js:fix` task before a commit is reverting always an uppercase `W` to `w`. I will take a look to disable that for this case.